jni/Android-nnstreamer.mk

```diff
@@ -83,10 +83,10 @@ $(foreach item,$(so_names_gst),$(eval $(call shared_lib_gst,$(item))))
 include $(CLEAR_VARS)
 
 # Please keep the pthread and openmp library for checking a compatibility
-LOCAL_ARM_NEON := true
-LOCAL_CFLAGS   += -O0 -DVERSION=\"$(NNSTREAMER_VERSION)\"
-LOCAL_CXXFLAGS += -std=c++11
-LOCAL_CFLAGS   += -pthread -fopenmp
+LOCAL_ARM_NEON      := true
+LOCAL_CFLAGS        += -O0 -DVERSION=\"$(NNSTREAMER_VERSION)\"
+LOCAL_CXXFLAGS      += -std=c++11
+LOCAL_CFLAGS        += -pthread -fopenmp
```

Comment:
unrelated changes?

Reply:
> 1. Fixed indentation
Need to be changed to keep same location with another statements.